Assets/Scripts/KatamariMovement.cs

```diff
@@ -342,196 +326,162 @@ void _PowerBar()
             go_up = true;
         }
 
-
         powerBar.GetComponent<Image>().fillAmount = power;
-
-
     }
 
     // Returns True if in golf hit mode
     public bool IsGolfHitMode()
     {
-        return myStateMachine == StateMachine.golfMode;
+        return movementState == StateMachine.golfMode;
     }
 
-    public void time_state_machine()
+    public void UpdateTimeStateMachine()
     {
-
-
         bool isFlyMovementDeadTime = resourceManager.GetTimeSinceLastHit() >= 0.75f;
```

Comment:
maybe we should make a public variable here so we can make fine tuning easier later. ALSO MY BAD XD

Reply:
we can at least make it a constant at the top of this file so it's easy to tune but won't get out of sync between scenes